plantcv/plantcv/watershed.py

```diff
@@ -75,8 +75,8 @@ def watershed_segmentation(rgb_img, mask, distance=10, label=None):
 
     # Reset color sequence mode
     params.color_sequence = color_sequence
-    _debug(visual=dist_transform,
-           filename=os.path.join(params.debug_outdir, str(params.device) + '_watershed_dist_img.png'),
+    _debug(visual=labels,
+           filename=os.path.join(params.debug_outdir, str(params.device) + '_watershed_labels_img.png'),
            cmap='gray')
     _debug(visual=joined,
            filename=os.path.join(params.debug_outdir, str(params.device) + '_watershed_img.png'))
```

Comment:
I think we should remove this debug image and instead should use [label2rgb ](https://scikit-image.org/docs/stable/api/skimage.color.html#skimage.color.label2rgb) to colorize the labeled mask for debug (rather than plotting the grayscale labels - like we do in `pcv.create_labels`).

Reply:
Good idea! I can implement these